driver/fixtures/fixtures_test.go

```diff
@@ -18,11 +18,11 @@ var Suite = &fixtures.Suite{
 	NewDriver: func() driver.BaseDriver {
 		return driver.NewExecDriverAt(filepath.Join(projectRoot, "native/exe/native"))
 	},
-	Transforms: driver.Transforms{
-		Native: normalizer.Native,
-		Code:   normalizer.Code,
-	},
+	Transforms: normalizer.Transforms,
 	BenchName: "class_complete",
+	Docker: fixtures.DockerConfig{
+		Image: "ruby:2.4",
+	},
```

Comment:
Please include the semantic config with a blacklist

Reply:
done.